proxy/default.conf.template

```diff
@@ -167,4 +167,14 @@ server {
 
         rewrite ^(.*)$ $senaite_rewrite/$1 break;
     }
-}
\ No newline at end of file
+}
+
+server {
+    listen 8082;
+    location / {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        set $erpnext erpnext:8080;
+        proxy_pass http://$erpnext;
```

Comment:
Will that work when ERPNext is not running?

Reply:
It should work since it's a valid Nginx configuration but on accessing the port localhost:8082 returns a Bad gateway because service `erpnext:8080` isn't available.

Reply:
Very good.